readthedocs/projects/views/private.py

```diff
@@ -510,15 +510,20 @@ def project_notifications(request, project_slug):
     webhook_form = WebHookForm(data=request.POST or None, project=project)
 
     if request.method == 'POST':
-        if email_form.is_valid():
-            email_form.save()
-        if webhook_form.is_valid():
-            webhook_form.save()
+        if 'email' not in request.POST:
+            email_form = EmailHookForm(data=None, project=project)
```

Comment:
Why is `data` None here? and also the code above is doing the same (lines ~509 - 510)

Reply:
Hi @stsewd, I have updated the code to display validation error for only the submitted form. Can you please review again?